desktop/linux/index.md

```diff
@@ -107,6 +101,60 @@ Alternatively, open a terminal and run:
 $ systemctl --user stop docker-desktop
 ```
 
+## Credentials management
+
+Docker Desktop relies on [`pass`](https://www.passwordstore.org/){: target="_blank" rel="noopener" class="_"} to store credentials in gpg2-encrypted files.
+Before signing in to Docker Hub from the Docker Dashboard or the Docker menu, you must initialize `pass`.
+Docker Desktop displays a warning if you've not initialized `pass`.
+
+You can intialize pass by using a gpg key. To generate a gpg key, run:
+
+``` console
+$ gpg --generate-key
+...
+GnuPG needs to construct a user ID to identify your key.
+
+Real name: Molly
+Email address: molly@docker.com
```

Comment:
We should use an `@example.com`  or `@example.org` domain for this; e-mail addresses `@docker.com` may actually land in some mail-boxes (and `example.com` is intended for this purpose.